packages/core/src/base/typography/link.scss

```diff
@@ -1,43 +1,61 @@
 /* stylelint-disable selector-class-pattern */
 @import '@cmsgov/design-system-support/src/settings/index';
 
-@if $ds-include-base-html-rulesets {
-  // <a> is the only base HTML element in the design system that
-  // has a style declaration. In all other cases, styles are applied using a
-  // namespaced class name. This selector isn't scoped under .ds-base, since that
-  // would cause the selector's specificity to be higher than most other component
-  // selectors (i.e. ds-c-button), necessitating overly specific selectors anytime
-  // a developer wanted to change an anchor's color property.
-  a {
-    color: $color-primary;
+%link {
+  color: $color-primary;
```

Comment:
I think this should also have a `text-decoration: underline;` and `cursor: pointer;`

Reply:
same with `%link-inverse`